src/ros_filter.cpp

```diff
@@ -408,12 +427,11 @@ namespace RobotLocalization
     }
 
     // Try to resolve the tf_prefix
-    nh_.param("/tf_prefix", tfPrefix_, std::string(""));
-
-    if (tfPrefix_.empty())
-    {
-      nh_.param("tf_prefix", tfPrefix_, std::string(""));
-    }
+    std::string tf_prefix_path;
```

Comment:
Please change this variable name to tfPrefixPath so as to maintain consistency with variable names